framework/db/Connection.php

```diff
@@ -607,6 +611,7 @@ protected function initConnection()
         if ($this->charset !== null && in_array($this->getDriverName(), ['pgsql', 'mysql', 'mysqli', 'cubrid'])) {
             $this->pdo->exec('SET NAMES ' . $this->pdo->quote($this->charset));
         }
+        $this->getSupportsResultSets();
```

Comment:
Yes by stopping db server before prepare/nextRowSet and see what happens :D
Although maybe other drivers may handle it differently? I will look into this.